### PR TITLE
Add what's new section for 0.15.1

### PR DESCRIPTION
**Pull request by SueChaplain**
Highlights for the 0.15 release with twitter
web intent tweet.

Signed-off-by: Sue Chaplain <sue_chaplain@uk.ibm.com>

### CODE REVIEW

**Comment by SueChaplain** [comment]
Jenkins website stage

File: oj9_whatsnew.html [hunk 1]
```diff
@@ -81,26 +81,72 @@ <h1>What's new?</h1>
         <span class="intro-text">Covering project news, events, milestones, and of course... new and cool stuff!</span>
       </div>
 
+      <div class="f-section-item" id="openj90150">
+      <div class="f-content-container">
+        <h3>Eclipse OpenJ9 version 0.15.0 released</h3>
```
Comment (pshipton) [inline oj9_whatsnew.html]: I'm thinking the 0.15.0 occurrences should be 0.15.1. Here, a couple lines down, and close to the end. If you get a download from Adopt labelled 0.15.1, you might expect the What's new to match.

Comment (pshipton) [reply oj9_whatsnew.html]: I've added a commit to fix it.

File: oj9_whatsnew.html [hunk 2]
```diff
@@ -81,26 +81,72 @@ <h1>What's new?</h1>
         <span class="intro-text">Covering project news, events, milestones, and of course... new and cool stuff!</span>
       </div>
 
+      <div class="f-section-item" id="openj90150">
```
Comment (keithc-ca) [inline oj9_whatsnew.html]: This and the reference below should probably use 'openj90151' instead of 'openj90150'.

Comment (pshipton) [reply oj9_whatsnew.html]: Fixed

**Pull request merged**